sources/packages/forms/src/form-definitions/studentfinancialinformationappeal.json

```diff
@@ -68,6 +68,25 @@
             "calculateValue": "// From the program year, get the calendar start year of the the program year.\r\n// e.g. program year  = 2022-2023, calendar start year is 2022. And tax year should be 2021.\r\nconst [startYear] = data.programYear ? data.programYear.split(\"-\"):[];\r\nconst taxYear = startYear ? (+startYear)-1 : \"\";\r\nvalue = taxYear;",
             "lockKey": true
         },
+        {
+            "input": true,
+            "tableView": true,
+            "key": "currentTaxYear",
+            "label": "currentTaxYear",
```

Comment:
Minor, the label can follow the same pattern as the exiting ones (e.g. `Current Tax Year`).
![image](https://github.com/user-attachments/assets/16819244-1e24-4ec0-a2eb-4e7f696109ec)